libclc/README.TXT

```diff
@@ -23,29 +23,41 @@ generic implementations of most library requirements, allowing the target
 to override the generic implementation at the granularity of individual
 functions.
 
-libclc currently only supports the PTX target, but support for more
+libclc currently supports NVPTX, AMDGPU and SPIRV targets, but support for more
```

Comment:
This is definitely better, but I'm not sure how we'd classify the `clspv` and `clspv64` targets as they don't conveniently fit into the "LLVM target" model. Same goes with SPIR-V, actually, as I pointed out further down. Saying "We support the SPIRV targets" might conflate the two notions of LLVM target and libclc target.

Reply:
I didn't add those mostly because I've got 0 idea what they are :)

Do you mean that someone may read this list and think that if libclc supports X then LLVM must have a target for X also? I wouldn't myself, but I'm far too biased to be useful here.

Perhaps instead of using the backend names (NVPTX) I should use ones from the folder names? amdgcn, ptx, spirv etc.

Reply:
(Especially if those names are what GPU folks are more likely to refer to them by)

Reply:
Done, tell me if those names are more appropriate.